Resources/Private/JavaScript/Host/index.js

```diff
@@ -44,7 +44,14 @@ document.addEventListener('DOMContentLoaded', () => {
     const translations = JSON.parse(appContainer.querySelector('[data-json="translations"]').innerHTML);
     const nodeTypeSchema = JSON.parse(appContainer.querySelector('[data-json="nodeTypeSchema"]').innerHTML);
     const reducers = reducerFactory(initialState);
-    const store = createStore(reducers);
+
+    const finalCreateStore = compose(
```

Comment:
Seems like compose is missing at this point. We have included `lodash.compose` in our dependencies already, so it needs to be imported here.

Anyway, great idea to include the dev tools :)